packages/table/src/interactions/selectable.tsx

```diff
@@ -181,7 +182,7 @@ export class DragSelectable extends React.Component<IDragSelectableProps, {}> {
         }
 
         const nextSelectedRegions = this.didExpandSelectionOnActivate
-            ? this.expandSelectedRegions(selectedRegions, region, focusedCell)
+            ? SelectionUtils.expandLastSelectedRegion(selectedRegions, region, focusedCell)
```

Comment:
Didn't _really_ need to move this code for this PR after all, but I like it in its new home so that you can see the parallelism of naming / possible parallelism of implementation / etc.